autoload/coquille.vim

```diff
@@ -2,7 +2,7 @@ let s:coq_running=0
 let s:current_dir=expand("<sfile>:p:h") 
 
 if !exists('coquille_auto_move')
-    let g:coquille_auto_move="false"
+    let g:coquille_auto_move="true"
```

Comment:
Please don't change things like this. Or at least propose them in another PR so they can be discussed independently.